django_project/base/templates/project/includes/project-panel.html

```diff
@@ -70,6 +70,11 @@ <h4 class="text-muted">
             </span>
           </p>
         {% endfor %}
+          {% if project.versions.count >= 2 %}
+              <strong><span class="text-muted">Latest 2 of {{ project.versions.count }} versions</span></strong>
```

Comment:
Ok if you look in the listing you will see I send more that two now. Anytime you are hard coding numbers like this it is a sign that we might be making a wrong turn. So better to soft code 2 here to the actual threshold and counts used.